src/site/includes/header.html

```diff
@@ -83,6 +83,10 @@
     window.VetsGov.headerFooter = {{ headerFooterData }};
   </script>
 
+
+  <script id="locizify" projectid="e096f2b1-ae77-47fd-9ef1-122884a31231" debug="false" apikey="a24af10f-8178-4e8f-bac1-76aebb982d78" saveMissing="true" referencelng="en" fallbacklng="en" src="https://unpkg.com/locizify@^4.0.7"></script>
```

Comment:
@youngfreezyVA Can we add and `async` attribute to this so it doesn't block page render?

Reply:
yeah for sure, will do. bigger picture question for you @timwright12 .. do you know why my `yarn.lock` needs to be updated, and all the linting jobs are failing when this is the only change i made and am up to date with master?

I think people are committing with `--no-verify` and its blocking this PR from getting a review instance :(

Reply:
Not completely sure, there's probably just a mismatch weirding out one of the tests TBH. I've seen it before and running `yarn` locally to update the lock file usually clears it out

Reply:
yeah i ran `yarn` in this branch, but then the lockfile shows a diff against master... meaning the lockfile in master is wrong

Reply:
it's ok if the lock files are different, they'll sync up on the next merge

Reply:
ok, thank you!